tests/unit/loopback_socket.hh

```diff
@@ -53,7 +53,7 @@ public:
     };
 private:
     bool _aborted = false;
-    queue<temporary_buffer<char>> _q{1};
+    queue<temporary_buffer<char>> _q{2};
```

Comment:
whats this for change do?

Reply:
Oh, turns out this causes some unit tests to hang.  I think this was written to expect that each packet would be read immediately, but with OpenSSL that isn't happening.  I saw in some tests that the server would write twice to the socket but the client side did not yet pick up the first packet so things would hang.  Incrementing this queue to 2 solved the issue.

Reply:
Oh and this loopback socket test thing just moves packages across memory, not over a network connection.